src/twisted/web/microdom.py

```diff
@@ -422,16 +421,16 @@ def cloneNode(self, deep=0, parent=None):
 
     def writexml(self, stream, indent='', addindent='', newl='', strip=0,
                  nsprefixes={}, namespace=''):
-        stream.write("<![CDATA[")
-        stream.write(self.nodeValue)
-        stream.write("]]>")
+        stream.write(u"<![CDATA[")
+        stream.write(u"" + self.nodeValue)
+        stream.write(u"]]>")
 
 def _genprefix():
     i = 0
     while True:
         yield  'p' + str(i)
         i = i + 1
-genprefix = _genprefix().next
+genprefix = next(_genprefix())
```

Comment:
`genprefix` used to be a function yielding the next prefix, but now this is just a value from the generator..

(this was supposed to go into the review i just submitted but i didn't actually click the "add comment" button :( )